pkg/distro/fedora/distro.go

```diff
@@ -233,20 +233,15 @@ var (
 	}
 
 	qcow2ImgType = imageType{
-		name:     "qcow2",
-		filename: "disk.qcow2",
-		mimeType: "application/x-qemu-disk",
+		name:        "qcow2",
+		filename:    "disk.qcow2",
+		mimeType:    "application/x-qemu-disk",
+		environment: &environment.KVM{},
```

Comment:
Shouldn't you also remove `cloud-init` from the image base package set? :thinking:

Reply:
It was never explicitly part of the base package sets in Fedora.  For the image types we modified here, it was always pulled in by `@Fedora Cloud Server`.